msu/systems/mod_settings/abstract_setting.nut

```diff
@@ -195,7 +195,7 @@
 	{
 		if (::MSU.Mod.Serialization.isSavedVersionAtLeast("1.2.0-rc.1", _in.getMetaData()))
 		{
-			this.__setFromSerializationTable(this.getMod().Serialization.flagDeserialize(format("MS.%s", this.getID())));
+			this.__setFromSerializationTable(this.getMod().Serialization.flagDeserialize(format("MS.%s", this.getID()), this.__getSerializationTable()));
```

Comment:
I'd prefer if we defined the format string either the line before or via a separate function so this line becomes more readable